tests/keyframes.rs

```diff
@@ -0,0 +1,20 @@
+#![cfg(test)]
+
+#[macro_use]
+mod macros;
+
+test!(
+    keyframes_basic_toplevel,
+    "@keyframes foo {\n  to {\n    a {\n      color: red;\n    }\n  }\n}\n"
+);
+error!(
+    keyframes_no_params,
```

Comment:
I believe keyframes *can* be declared without params

Reply:
Yes, haven't fixed that.